### DISCUSSION TITLE
Default library restriction on user creation

### DISCUSSION THREAD

**Discussion by DuvelCorp** [gotson/komga#1231] [Ideas]
On user creation, Is there anyway to change the default komga behavior for library access, which is "Allow all"  ?
I'd like to be able to change that to to only the libraries of my choice, or to exclude some libraries by default.

Use case is : I do have large libraries for children, and other for adults. And also some xxx comic books and magazines libraries which are obviously touchy and I dont want to share this by default, and  only grant access to them to a very few users.

When I create users manually I always have to uncheck default and grant access to each library manually. Its not that much an issue for a manual user creation process.

However I understand that if I setup Oauth2 automatic user creation (with Authentik), I wont have any control on that, and that I will have to manually go in Komga User management to change the library access AFTER the user was created automatically by a successful first Oauth2 connection. And it is clear that this wont work on the long term because I have dozens of users and several admins that can manage them

Cheers

**Comment by gotson** [comment]
Duplicate of https://github.com/gotson/komga/issues/947
